plugins/node/opentelemetry-instrumentation-aws-sdk/src/services/MessageAttributes.ts

```diff
@@ -64,7 +64,7 @@ export const injectPropagationContext = (
   attributesMap?: SQS.MessageBodyAttributeMap | SNS.MessageAttributeMap
 ): SQS.MessageBodyAttributeMap | SNS.MessageAttributeMap => {
   const attributes = attributesMap ?? {};
-  if (Object.keys(attributes).length < MAX_MESSAGE_ATTRIBUTES) {
+  if (Object.keys(attributes).length + propagation.fields().length <= MAX_MESSAGE_ATTRIBUTES) {
```

Comment:
can you please add a simple unit test for that?

Reply:
@blumamir  👍
[Added tests](https://github.com/open-telemetry/opentelemetry-js-contrib/pull/738/commits/75d653f6541ea0cb30ac1df39fb8db2456e338e4)